### DISCUSSION TITLE
[Question]: Can I change a embedding model?

### DISCUSSION THREAD

**Discussion by angpao** [danny-avila/LibreChat#4729] [Q&A]
### What is your question?

Can I change a embedding model from azure open ai text-embedding-3-small to Gemini API  models/text-embedding-004?

### More Details

-

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Check out the readme: https://github.com/danny-avila/rag_api

**Comment by angpao** [comment]
**Not include gemini,right?** 
EMBEDDINGS_PROVIDER: (Optional) either "openai", "bedrock", "azure", "huggingface", "huggingfacetei" or "ollama", where "huggingface" uses sentence_transformers; defaults to "openai"

EMBEDDINGS_MODEL: (Optional) Set a valid embeddings model to use from the configured provider.

Defaults
openai: "text-embedding-3-small"
azure: "text-embedding-3-small" (will be used as your Azure Deployment)
huggingface: "sentence-transformers/all-MiniLM-L6-v2"
huggingfacetei: "http://huggingfacetei:3000/". Hugging Face TEI uses model defined on TEI service launch.
ollama: "nomic-embed-text"
bedrock: "amazon.titan-embed-text-v1"
